sender.go

```diff
@@ -52,9 +62,34 @@ var gcmSendEndpoint = GcmSendEndpoint
 //	}
 type Sender struct {
 	ApiKey string
+	URL    string
 	Http   *http.Client
 }
 
+// NewClient returns a new sender with the given URL and apiKey.
+// If one of input is empty or URL is malformed, returns error.
+// It sets http.DefaultHTTP client for http connection to server.
+// If you need our own configuration overwrite it.
+func NewClient(urlString, apiKey string) (*Sender, error) {
+	if len(urlString) == 0 {
+		return nil, fmt.Errorf("missing GCM/FCM endpoint url")
+	}
+
+	if len(apiKey) == 0 {
+		return nil, fmt.Errorf("missing API Key")
+	}
+
+	if _, err := url.Parse(urlString); err != nil {
+		return nil, fmt.Errorf("failed to parse URL %q: %s", urlString, err)
+	}
+
+	return &Sender{
+		URL:    urlString,
+		ApiKey: apiKey,
+		Http:   http.DefaultClient,
```

Comment:
- It's not related to this PR (by default, this package use default client)
- (even if it's related) It's not what package author should handle. Since `Http` filed is exported, user can replace it with their own HTTP client with their favorite timeout or what ever. See other client package, most of them use default client and just allow to modify it on user side.